src/states_screens/race_gui_base.cpp

```diff
@@ -99,6 +99,7 @@ RaceGUIBase::RaceGUIBase()
     m_gauge_full_bright     = irr_driver->getTexture(file_manager->getAsset(FileManager::GUI_ICON,"gauge_full_bright.png"));
     m_gauge_empty           = irr_driver->getTexture(file_manager->getAsset(FileManager::GUI_ICON,"gauge_empty.png"));
     m_gauge_goal            = irr_driver->getTexture(file_manager->getAsset(FileManager::GUI_ICON,"gauge_goal.png" ));
+    m_lap_flag              = irr_driver->getTexture(FileManager::GUI_ICON, "lap_flag.png");
```

Comment:
Any reason why this is different than the one above it?

Reply:
Yes, I moved that line from race_gui.cpp to here. The method has different overloads, so I guess the lines above could be simplified as well but didn't test that. I thought changing them shouldn't be part off this PR.